doc/_toc.yml

```diff
@@ -37,6 +37,7 @@ parts:
 
   - caption: Community
     chapters:
+    - file: community/coc
+    - file: community/support
     - file: community/projects
-    - file: community/community
```

Comment:
Why are we switching it from community to support?
code of conduct should be part of it too.

Reply:
I guess I want to have consistent with how we did in [Ploomber doc](https://docs.ploomber.io/en/latest/community/index.html)

<img width="248" alt="Screenshot 2023-01-30 at 4 28 04 PM" src="https://user-images.githubusercontent.com/123580782/215599024-857d0e48-2056-4d7a-9ba8-94f30df37769.png">

Do you think we should "standardize" the pattern among all our projects? e.g. jupyter, sklearn-evaluation, ploomber, should have the same doc structure like:
[Community](https://docs.ploomber.io/en/latest/community/index.html#)
- [Contact Us](https://docs.ploomber.io/en/latest/community/contact.html)
- [Code of Conduct](https://docs.ploomber.io/en/latest/community/coc.html)
- [API Changes](https://docs.ploomber.io/en/latest/community/api-changes.html)
- [Contributing](https://docs.ploomber.io/en/latest/community/contributing.html)
- [User Statistics](https://docs.ploomber.io/en/latest/community/user-stats.html)
- [Roadmap](https://docs.ploomber.io/en/latest/community/roadmap.html)

Reply:
@tonykploomber those were nested inside the community section, here you simply removed it and flatten?
@edublancas thoughts?

Reply:
i think it's clearer this way. let's keep this change